mautic/mautic
# [UX] Change "Visible on short forms" to "Visible on quick add"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR introduces a terminology update to unify the language across the platform. The label “Visible on short forms” has been revised to “Visible on quick add,” which is consistent with the terms used in other sections. This change promotes a cohesive user experience by standardizing the language, making it easier for users to understand and navigate the platform. It is especially beneficial for users who interact with various forms and features, ensuring a seamless and intuitive interface.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Custom fields
3. Create new
4. Look at the label

![image](https://github.com/mautic/mautic/assets/116097999/8e8d9e9a-5e78-45ad-aa9a-87c8057747fe)
